gitops-v2/build/main.yaml

```diff
@@ -94,6 +101,26 @@ stages:
               condition: and(succeeded(), eq(${{ parameters.dockerLint.enable }}, true))
           - bash: |
               set -ex
+              # Download
+              wget -O horusec https://github.com/ZupIT/horusec/releases/download/${HORUSEC_TAG}/horusec_linux_x86
+
+              DOWNLOAD_HORUSEC_SHA=$(openssl sha1 -sha256 horusec | awk '{print $2}')
+              if [[ "${HORUSEC_SHA}" != "${DOWNLOAD_HORUSEC_SHA}" ]]; then
+                  echo "Downloaded checksum (${DOWNLOAD_HORUSEC_SHA}) for hadolint does not match expected value: ${HORUSEC_SHA}"
+                  exit 1
+              fi
+              chmod +x horusec
+              # Run
+              ./horusec start -p . -e=true
+            displayName: Horusec
+            continueOnError: ${{ parameters.horusec.ignoreRuleViolations }}
+            env:
+              HORUSEC_TAG: ${{ parameters.binaries.horusec.tag }}
+              HORUSEC_SHA: ${{ parameters.binaries.horusec.sha }}
+            ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
```

Comment:
I think you want something like this:

https://github.com/XenitAB/azure-devops-templates/blob/4d6c0df22cda97dea5222bd91bf4fdb839376760/gitops/build/main.yaml#L90-L93